lib/phoenix_html/link.ex

```diff
@@ -165,4 +175,11 @@ defmodule Phoenix.HTML.Link do
 
     {value, opts}
   end
+
+  defp invalid_destination?(to) when is_tuple(to), do: false
+
+  for scheme <- @valid_uri_schemes do
+    defp invalid_destination?(unquote(scheme) <> _), do: false
+  end
+  defp invalid_destination?(to), do: String.contains?(to, ":")
```

Comment:
We probably need a `is_binary(to)` guard?

Reply:
Yep!